crates/fluent-check/src/main.rs

```diff
@@ -0,0 +1,213 @@
+use std::{
+    collections::{HashMap, HashSet},
+    fs::File,
+    io::Read,
+    path::Path,
+    process::Command,
+};
+
+use fish_build_helper::workspace_root;
+
+use fluent::{FluentBundle, FluentResource};
+use unic_langid::LanguageIdentifier;
+
+fn main() {
+    let unique_ids = extract_fluent_ids();
+
+    let ftl_dir = workspace_root().join("localization").join("fluent");
```

Comment:
nit:
it might be better to replace "ftl" with "translation_list" in places where "fluent" is implied by context,
and use "Fluent Translation List" elsewhere.
But no need to spend time on that, especially since these are all private AFAICT.